tests/integration/devfile/cmd_logs_test.go

```diff
@@ -12,6 +13,21 @@ var _ = Describe("odo logs command tests", func() {
 	var componentName string
 	var commonVar helper.CommonVar
 
+	areAllPodsRunning := func() bool {
+		allPodsRunning := true
+		status := string(commonVar.CliRunner.Run("get", "pods", "-n", commonVar.Project, "-o", "jsonpath=\"{.items[*].status.phase}\"").Out.Contents())
+		// value of status would be a string decorated by double quotes; so we ignore the first and last character
+		// this could have been done using strings.TrimPrefix and strings.TrimSuffix, but that's two lines/calls.
+		status = status[1 : len(status)-1]
```

Comment:
You can use strings.Trim(), which is a single call. I would prefer as it will also work if there are not double quotes in the output (and I cannot see them on my side):

```
$ kubectl get pods -n olm -o jsonpath="{.items[*].status.phase}"
Succeeded Running Succeeded Running Running Running Running
```

Reply:
> You can use strings.Trim(), which is a single call.

👍🏾 

> (and I cannot see them on my side)

Neither do I except when you run this thing in a Go program. It took a good deal of my time because string comparison between `"Running` and `Running` or `Running"` and `Running` obviously yielded `false`. 😄